package/doc/sphinx/source/conf.py

```diff
@@ -68,7 +69,7 @@
 # (take the list from AUTHORS)
 # Ordering: (1) Naveen (2) Elizabeth, then all contributors in alphabetical order
 #           (last) Oliver
-author_list = __import__('MDAnalysis').__authors__
+author_list = mda.__authors__
```

Comment:
Removing `__import__` was necessary for RTD to import MDAnalysis properly without dying because it couldn't find the C extensions.

Reply:
Well, if this works... no idea how this is different from `__import__()`.